### PR TITLE
Updated publish date format to be the same as fe-blogs

### PR DESCRIPTION
**Pull request by danielmoll**
This is reliant on a change to the api data from WIN website. PR in motion!

I also took this opportunity to address some of the previous PR points re the example not reflecting real use. I've updated the example to use `component-react-async-container` as the WIN website uses it. The data remains stubbed, but is now handled by the assets script and lives in folder named `stubbed`.

I know this doesn't address everything but it's step towards it.

### CODE REVIEW

File: src/example.js [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-import article from '../example-data/article';
+import fetch from 'isomorphic-fetch';
+
+function fetchArticle() {
+  return fetch('/stubbed/article.json')
+    .then((response) => (response.json()));
+}
+
+function handleLoading() {
+  return (<div>Loading...</div>);
```
Comment (umbobabo) [inline src/example.js]: I think we have a component for handle loading and it already used in the Re-Vamp so we could align it here.

**Comment by umbobabo** [comment]
Just a NTH, but I'm happy to merge anyway.

File: src/subheader-leader.js [hunk 2]
```diff
@@ -11,18 +11,18 @@ export default function WinLeaderSubheader({
...
+        dateTime={publishDate.raw}
         className={[
           ...generateClassNameList('article-template__pubdate'),
           ...extendedSubheaderItemClasses,
         ].join(' ')}
       >
-        {formattedPublishDate}
-      </h2>
+        {publishDate.formatted}
+      </time>
```
Comment (keithamus) [inline src/subheader-leader.js]: 👍

**Comment by keithamus** [comment]
LGTM. I'll leave @umbobabo to merge considering his above comments.

**Comment by danielmoll** [comment]
@umbobabo updated with `component-loading` - is there a component for handling error/failure messages?

**Comment by umbobabo** [comment]
Unfortunately not at the moment. I pushed different times to the Designers/UX of the re-Vamp to start having a basic messagging system to use across projects for error. But nobody listen me.

**Comment by danielmoll** [comment]
Hmmm... just seen this is no longer passing tests... Let me see what's happening with this.

**Comment by danielmoll** [comment]
@umbobabo all good now. It was the ad component in the stubbed data causing a global leak in the tests causing them to error behind the scenes.

**Comment by keithamus** [comment]
LGTM again

**Pull request merged**